app/views/waivers/new.html.erb

```diff
@@ -1,108 +1,103 @@
-
-
-<% if !current_user.participant.is_scc? or params[:participant_id]== nil %>
 <div class="page-header">
   <h1>Midway Safety Video</h1>
 </div>
+
 <p>You must watch the video prior to signing the waiver.</p>
 
 <div id="player"></div>
 <div id="waiver" style="display: none;">
-  <% else %>
-  <div id="waiver">
-<% end %>
-
-
-  <h2><%= Time.now.year %> SPRING CARNIVAL MIDWAY RELEASE OF LIABILITY</h2>
-
-  <p>
-
-  I, <%= @user.name %>, want to participate in Carnegie Mellon University (“CMU”) <%= Time.now.year %> Spring Carnival Midway construction
-
-  activities and/or be present on the Midway construction site (collectively, the “Activities.”) I understand
-
-  that the Activities may involve climbing, crawling, physical exertion, the use of and/or proximity to hand
-
-  and power tools and construction materials and equipment.
-  </p>
-  <p>
-
-  I understand that there are risks (including, but not limited to, the risk of injury, disability and death)
-
-  associated with my participation in the Activities. I voluntarily assume all the risks. In consideration of
 
-  the benefits of participation in the Activities, I hereby, on behalf of myself and those acting on my
+<h2><%= Time.now.year %> SPRING CARNIVAL MIDWAY RELEASE OF LIABILITY</h2>
 
-  behalf, irrevocably and unconditionally release, waive and promise not to sue CMU from/for any and all
+<p>
```

Comment:
This clean up had to do with changes in how the video is shown (`@should_show_video`) so it was included. Other refactoring will be done on new branches.